parcel_test.go

```diff
@@ -31,75 +31,108 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
-	store := NewParcelStore(db)
-	parcel := getTestParcel()
+	db, err := sql.Open("sqlite", "tracker.db") // подключение к БД
+	require.NoError(t, err)                     // проверка ошибки
+	store := NewParcelStore(db)                 // создание тестовой посылки
+	parcel := getTestParcel()                   // получение тестовой посылки
+	defer db.Close()
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
-
+	parcel.Number, err = store.Add(parcel)
+	require.NoError(t, err)
+	require.NotEmpty(t, parcel.Number)
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
-
+	checkParcel, err := store.Get(parcel.Number)
+	require.NoError(t, err)
+	require.Equal(t, parcel, checkParcel)
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
+	err = store.Delete(parcel.Number)
+	require.NoError(t, err)
+
 	// проверьте, что посылку больше нельзя получить из БД
+	_, err = store.Get(parcel.Number)
+	require.Equal(t, sql.ErrNoRows, err)
 }
 
 // TestSetAddress проверяет обновление адреса
 func TestSetAddress(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
-
+	db, err := sql.Open("sqlite", "tracker.db") // настройте подключение к БД
+	store := NewParcelStore(db)                 // создание тестовой посылки
+	parcel := getTestParcel()                   // получение тестовой посылки
+	require.NoError(t, err)                     // проверка ошибки
+	defer db.Close()
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
-
+	parcel.Number, err = store.Add(parcel)
+	require.NoError(t, err)
+	require.NotEmpty(t, parcel.Number)
 	// set address
 	// обновите адрес, убедитесь в отсутствии ошибки
 	newAddress := "new test address"
-
+	err = store.SetAddress(parcel.Number, newAddress)
+	require.NoError(t, err)
 	// check
 	// получите добавленную посылку и убедитесь, что адрес обновился
+	checkParcel, err := store.Get(parcel.Number)
+	require.NoError(t, err)
+	require.Equal(t, checkParcel.Address, newAddress)
```

Comment:
На втором месте нужно ставить что хотим увидеть, а на третьем, что проверяем.
На самом тест это не влияет, но когда он упадет, то вывод будет неправильный
```suggestion
	require.Equal(t, newAddress, checkParcel.Address)
```